qa/tasks/vstart_runner.py

```diff
@@ -773,6 +766,29 @@ def list_connections():
 
         self.gather_mount_info()
 
+    def _set_fuse_daemon_pid(self):
+        # NOTE: When a command <args> is launched with sudo, two processes are
+        # launched, one with sudo in <args> and other without. Make sure we
+        # get the PID of latter one.
+        for i in range(7):
+            try:
+                sock = self.find_admin_socket()
+            except RuntimeError:
+                pass
+
+            try:
+                pid = re.match(".*\.(\d+)\.asok$", f).group(1)
```

Comment:
please s/f/sock/

Reply:
IMO, for the loop, it should be something like:

```
 775     def _set_fuse_daemon_pid(self):
 776         # NOTE: When a command <args> is launched with sudo, two processes are
 777         # launched, one with sudo in <args> and other without. Make sure we
 778         # get the PID of latter one.        
 779         for i in range(7):
 780             try:
 781                 sock = self.find_admin_socket()     
 782                 try:
 783                     pid = re.match(".*\.(\d+)\.asok$", sock).group(1)
 784                     self.fuse_daemon.subproc.pid = int(pid)
 785                     return
 786                 except Exception as e:              
 787                     print(e)
 788             except RuntimeError:                                                                                                      
 789                 pass
 790   
 791             time.sleep(2)
 792         else:
 793             raise RuntimeError('Failed to get the PID of ceph-fuse process '
 794                                'not containing sudo in args.')     
 795   
```

Reply:
There's no need for nested try-except, catching for CommandFailedError must fix it.